3p/twitter.js

```diff
@@ -49,13 +49,17 @@ export function twitter(global, data) {
     delete data.height;
     twttr.widgets.createTweet(data.tweetid, tweet, data)./*OK*/then(() => {
       const iframe = global.document.querySelector('#c iframe');
-      // Unfortunately the tweet isn't really done at this time.
-      // We listen for resize to learn when things are
-      // really done.
-      iframe.contentWindow.addEventListener('resize', function() {
+      // There is no iframe if the tweet was deleted. Thanks for resolving
+      // the promise, though :)
+      if (iframe && iframe.contentWindow) {
+        // Unfortunately the tweet isn't really done at this time.
+        // We listen for resize to learn when things are
+        // really done.
+        iframe.contentWindow.addEventListener('resize', function() {
```

Comment:
probly could just get rid of the extra function and pass in `render`, but since we dont have tests probably best to leave it for now :cat: